.travis.yml

```diff
@@ -12,6 +17,15 @@ install:
 script:
   - ./scripts/pre-commit.sh
   - vendor/github.com/kubernetes/repo-infra/verify/verify-boilerplate.sh -- rootdir="$GOPATH/src/github.com/kubernetes-sigs/federation-v2" -v
+deploy:
+  skip_cleanup: true
+  provider: script
+  script: imagebuild.sh
```

Comment:
would you put the `imagebuild.sh` script under the `scripts` dir

Reply:
Sure.